test/extensions/reputation-bootstrapper.js

```diff
@@ -165,14 +166,6 @@ contract("Reputation Bootstrapper", (accounts) => {
       await checkErrorRevert(reputationBootstrapper.claimGrant(PIN1, { from: USER1 }), "reputation-bootstrapper-nothing-to-claim");
     });
 
-    it("cannot claim reputation amounts and tokens if the token amount only partially covers the balance", async () => {
```

Comment:
Shouldn't this still be the case?

Reply:
Yes, I deleted it because we'd only be checking the underlying underflow math for token transfers, which I didn't think was particularly interesting, but I can put it back.

Reply:
That's ultimately the error that throws, but we're interested in checking the logic that if that does end up throwing, then we don't also award reputation.